core/include/traccc/definitions/primitives.hpp

```diff
@@ -32,6 +32,7 @@ using channel_id = unsigned int;
 // Default algebra type
 using default_algebra = ALGEBRA_PLUGIN<traccc::scalar>;
 
+using size_type = detray::dsize_type<default_algebra>;
```

Comment:
I know why algebras need a size type. But I'm not super happy about using a global `traccc::size_type`. 🤔 We will still have different size types here and there. The `traccc::size_type` name, I fear, will be too misleading.

Reply:
Ok, I just tried replicate what I found in the IO code